### PR TITLE
American journal of climate change

### PR DESCRIPTION
**Pull request by arunbte**
Citation style for the journal American Journal of Climate Change

### CODE REVIEW

**Comment by rmzelle** [comment]
@arunbte, please update your style according to https://github.com/citation-style-language/styles/wiki/Style-Requirements. Let us know if you need help.

**Comment by adam3smith** [comment]
(let us know if you need help; this is currently an empty submission)

File: american-journal-of-climate-change.csl [hunk 1]
```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>American Journal of Climate Change</title>
+    <title-short>AJCC</title-short>
+    <id>http://www.zotero.org/styles/american-journal-of-climate-change</id>
+	<link href="http://www.zotero.org/styles/american-journal-of-climate-change" rel="self"/>
+	<issn>2167-9495</issn>
+      <eissn>2167-9509</eissn>
+	<link href="http://www.scirp.org/journal/ajcc/" rel="documentation"/>
```
Comment (adam3smith) [inline american-journal-of-climate-change.csl]: which CSL style did you base this on? Please add with link `rel="template"`

File: american-journal-of-climate-change.csl [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      <contributor>
+      <name>Arunkumar R</name>
+      <uri>http://www.mendeley.com/profiles/arunkumar-r1/</uri>
+    </contributor>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
```
Comment (adam3smith) [inline american-journal-of-climate-change.csl]: please use something less generic ;). Full list of options is here: http://docs.citationstyles.org/en/stable/specification.html#appendix-i-categories
I'd guess `science` and/or `geology` would work.

File: american-journal-of-climate-change.csl [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        </if>
+        <else-if type="chapter paper-conference" match="any">
+          <text variable="title" suffix=". "/>
+          <text term="in" suffix=": "/>
+          <text macro="editor" suffix=", "/>
+          <text variable="container-title" form="short" text-case="title" suffix=", "/>
+          <text macro="edition" suffix=", "/>
+          <text macro="publisher"/>
+          <group delimiter=" ">
+            <label variable="page" form="short" prefix=": "/>
```
Comment (adam3smith) [inline american-journal-of-climate-change.csl]: this seems wrong: the publisher macro ends with a period, i.e. you're getting `place. : pp.` here, which I can't imagine being correct.

File: american-journal-of-climate-change.csl [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+            <text variable="publisher"/>
+            <text macro="year-date"/>
+          </group>
+        </else-if>
+        <else>
+          <group delimiter=" ">
+            <text variable="title" suffix="."/>
+            <text variable="container-title" text-case="title" font-style="italic" suffix="."/>
+            <text variable="volume" font-weight="normal"/>
+            <text variable="issue" prefix="(" suffix="), "/>
```
Comment (adam3smith) [inline american-journal-of-climate-change.csl]: remove the space in the suffix (which is already in the delimiter).

File: american-journal-of-climate-change.csl [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <text variable="publisher" suffix=", "/>
+    <text variable="publisher-place" suffix=".  "/>
```
Comment (adam3smith) [inline american-journal-of-climate-change.csl]: I actually think this period needs to go. Book chapter end with "place, 12-14", which this prevents

File: american-journal-of-climate-change.csl [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
+    <summary>A style for American Journal of Climate Change that includes article titles in the reference list</summary>
+    <updated>2015-12-16T21:10:23+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author" suffix=" ">
+      <name and="text" initialize-with="." name-as-sort-order="all"/>
+      <label form="short" prefix=", "/>
```
Comment (adam3smith) [inline american-journal-of-climate-change.csl]: `text-case="capitalize-first"`

File: american-journal-of-climate-change.csl (hunk 2 above)
Comment (arunbte) [inline american-journal-of-climate-change.csl]: Changed to `science`

**Comment by adam3smith** [comment]
See comments in line. Please makes sure that, at a minimum, citations are correctly formatted for journal articles, book chapters, and books -- I only went through this quickly, not closely enough to catch all issues. Instructions for updating the style are in the automated message above. Any questions, let us know. 
(Just as a warning -- I'm no expert, but SCIRP doesn't have a great reputation as a scientific publisher. You're sure you want to submit with them? See e.g. http://boingboing.net/2012/10/19/math-journal-accepts-computer.html)

**Comment by rmzelle** [comment]
@arunbte, we can accept this once you address @adam3smith's requests.

Also please visit https://github.com/arunbte/styles/blob/american-journal-of-climate-change/water-resources-research.csl and delete the WRR style from this pull request via the trash icon.

File: american-journal-of-climate-change.csl (hunk 1 above)
Comment (rmzelle) [inline american-journal-of-climate-change.csl]: @arunbte, if you know which style you used as a template, please add a "template" link. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link

File: american-journal-of-climate-change.csl (hunk 5 above)
Comment (rmzelle) [inline american-journal-of-climate-change.csl]: `suffix=".  "` also contains a double space, which can't be correct.

**Comment by rmzelle** [comment]
@arunbte, thanks.

Could you address the remaining comments from @adam3smith?:

* https://github.com/citation-style-language/styles/pull/2056/files#r63588644
* https://github.com/citation-style-language/styles/pull/2056/files#r63590015
* https://github.com/citation-style-language/styles/pull/2056/files#r63589974
* https://github.com/citation-style-language/styles/pull/2056/files#r63589250
* https://github.com/citation-style-language/styles/pull/2056/files#r63589309

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**